### PR TITLE
ci: update test configuration

### PR DESCRIPTION
**Pull request by ikuleshov**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-analytics-admin/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/noxfile_config.py [hunk 1]
```diff
@@ -1,20 +1,15 @@
 TEST_CONFIG_OVERRIDE = {
-    # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7"],
```
Comment (parthea) [inline samples/noxfile_config.py:3]: Python 2.7 is ignored by default. I've deleted this line.

File: samples/noxfile_config.py [hunk 2]
```diff
@@ -1,20 +1,15 @@
...
-    # Old samples are opted out of enforcing Python type hints
-    # All new samples should feature them
-    "enforce_type_hints": True,
```
Comment (parthea) [inline samples/noxfile_config.py:6]: The comment above does not reflect the current state as the existing samples are not opted out. I've deleted this line. If we want to add this setting, it should be done in a separate PR.

File: samples/properties_firebase_links_list.py [hunk 3]
```diff
@@ -22,7 +22,6 @@
 """
 # [START analyticsadmin_properties_firebase_links_list]
 from google.analytics.admin import AnalyticsAdminServiceClient
-from google.analytics.admin_v1alpha.types import MaximumUserAccess
```
Comment (parthea) [inline samples/properties_firebase_links_list.py:25]: `MaximumUserAccess` was removed from the protos. See #119

**Comment by parthea** [comment]
@googlebot I consent.

**Comment by parthea** [comment]
@ikuleshov Please could you take a look?

```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
properties_user_links_list.py:39: in list_property_user_links
    results = client.list_user_links(parent=f"properties/{property_id}")
../google/analytics/admin_v1alpha/services/analytics_admin_service/client.py:1574: in list_user_links
    response = rpc(request, retry=retry, timeout=timeout, metadata=metadata,)
.nox/py-3-8/lib/python3.8/site-packages/google/api_core/gapic_v1/method.py:142: in __call__
    return wrapped_func(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

args = (parent: "properties/276206997"
,)
kwargs = {'metadata': [('x-goog-request-params', 'parent=properties/276206997'), ('x-goog-api-client', 'gl-python/3.8.8 grpc/1.39.0 gax/2.0.1 gapic/0.5.0')]}

    @functools.wraps(callable_)
    def error_remapped_callable(*args, **kwargs):
        try:
            return callable_(*args, **kwargs)
        except grpc.RpcError as exc:
>           raise exceptions.from_grpc_error(exc) from exc
E           google.api_core.exceptions.PermissionDenied: 403 The caller does not have permission

.nox/py-3-8/lib/python3.8/site-packages/google/api_core/grpc_helpers.py:68: PermissionDenied
______ test_properties_web_data_streams_get_enhanced_measurement_settings ______
```

**Comment by parthea** [comment]
@ikuleshov Friendly ping

**Comment by ikuleshov** [comment]
@parthea Looks like all tests are passing now!

**Review by parthea — APPROVED** [review]
Thanks @ikuleshov !
